spec.md

```diff
@@ -115,6 +116,19 @@ Return a list of counties in `Division` data structure.
 A `prefecture_code` is a 6-length code that endswith `00`. It can also be a 4-length code.
 
 
+### `#revisions`
+
+List all revisions supported by GB2260. This is a module(class) constant.
+
+
+### `#search(code, str[] revisions)`
```

Comment:
It seems that the second argument is not necessary. I think the `search` method should return a list of matched items. 

The callee users could pick a subset by themselves. Then this interface will be single-responsibility.